sdk/typescript/test/e2e/dynamic-fields.test.ts

```diff
@@ -18,12 +19,12 @@ describe('Dynamic Fields Reading API', () => {
       .getOwnedObjects({
         owner: toolbox.address(),
         options: { showType: true },
+        filter: { StructType: `${packageId}::dynamic_fields_test::Test` },
       })
       .then(function (objects) {
-        const obj = objects.filter(
-          (o) => o.type === `${packageId}::dynamic_fields_test::Test`,
-        );
-        parentObjectId = obj[0].objectId;
+        console.log(objects);
```

Comment:
@healthydeve Looks like this snuck in

Reply:
https://github.com/MystenLabs/sui/pull/9494

Reply:
oops. Here is the removal.